app/models/federated_protocols/saml.rb

```diff
@@ -31,8 +31,11 @@ def vtr
     end
 
     def requested_attributes
-      @requested_attributes ||= SamlRequestPresenter.new(
-        request: request, service_provider: current_service_provider,
+      @requested_attributes ||= SamlRequestedAttributesPresenter.new(
+        service_provider: current_service_provider,
+        ial: ial,
```

Comment:
The `ial` method here applies the SP default `ial` value if no ACR value is provided. For this reason we do not need to consider `service_provider.ial` in this class anywhere.

https://github.com/18F/identity-idp/blob/7ec7182daf09a87dffc5090a67ddf636b586689d/app/models/federated_protocols/saml.rb#L13-L19

https://github.com/18F/identity-idp/blob/7ec7182daf09a87dffc5090a67ddf636b586689d/app/models/federated_protocols/saml.rb#L51-L57